Source/Core/DolphinQt2/GameList/GameListProxyModel.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Resources.h"
+#include "DolphinQt2/GameList/GameListModel.h"
+#include "DolphinQt2/GameList/GameListProxyModel.h"
+
+static const QSize NORMAL_BANNER_SIZE = QSize(96, 32);
```

Comment:
These can be made constexpr.

e.g.

```cpp
static constexpr QSize NORMAL_BANNER_SIZE(96, 32);
static constexpr QSize LARGE_BANNER_SIZE(144, 48);
```
or (with uniform initialization syntax)

```cpp
static constexpr QSize NORMAL_BANNER_SIZE{96, 32};
static constexpr QSize LARGE_BANNER_SIZE{144, 48};
```